packages/typescript/ai-client/tests/chat-client.test.ts

```
@@ -8,7 +8,10 @@ import {
   createApprovalToolCallChunks,
   createCustomEventChunks,
 } from './test-utils'
-import type { ConnectionAdapter } from '../src/connection-adapters'
+import type {
+  ConnectionAdapter,
+  ConnectConnectionAdapter,
```

ESLint

[error] Member 'ConnectConnectionAdapter' of the import declaration should be sorted alphabetically.

(sort-imports)